src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__day calendar__day--1"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
Можна було використати nth-child замість того, щоб створювати безліч класів